packages/dataviews/src/field-types/email.tsx

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
Should we extract the email field type addition to its own PR for better reviews.